spec/factories/state_file_nj_intakes.rb

```diff
@@ -110,5 +111,21 @@
     trait :df_data_many_w2s do
       raw_direct_file_data { StateFile::XmlReturnSampleService.new.read('nj_zeus_many_w2s') }
     end
+
+    trait :married do
+      filing_status { "married_filing_jointly" }
+      spouse_birth_date { Date.new(1990, 1, 1) }
+      spouse_ssn { "123456789" }
+    end
+
+    trait :primary_over_65 do
+      primary_birth_date { Date.new(1900, 1, 1) }
```

Comment:
[pebble] Here and in the other tests, I would recommend using a test year that is on the edge of 65: 1959. Otherwise you're not testing the "over 65" logic that you wrote and the tests would still pass even if we accidentally flubbed our logic in the code and made it so over 55 still gets checked or something.